pyproject.toml

```diff
@@ -0,0 +1,100 @@
+[tool.poetry]
+name = "mozilla-bigquery-etl"
+version = "20.12.2"
+description = "Tooling for building derived datasets in BigQuery"
+homepage = "https://github.com/mozilla/bigquery-etl"
+authors = ["Mozilla Corporation <fx-data-dev@mozilla.org>"]
+packages = [
+  { include = "bigquery_etl" },
+  { include = "sql_generators" },
+]
+include = ["target/*.jar"]
+
+[tool.poetry.scripts]
+bqetl = 'bigquery_etl.cli:cli'
+
+
+[tool.poetry.dependencies]
+python = "^3.10,<3.11"
```

Comment:
unless poetry doesn't support it, this should probably use `~=3.10`, which means the same thing

Reply:
I don't think poetry supports that, at least it thinks the python constraint is `>=3.10,<4.0` when i set it to `~=3.10`